DataFormats/EcalDigi/src/EcalDataFrame_Ph2.cc

```diff
@@ -0,0 +1,36 @@
+#include "DataFormats/EcalDigi/interface/EcalDataFrame_Ph2.h"
+#include "CondFormats/EcalObjects/interface/EcalConstants.h"
+#include <iostream>
+
+int EcalDataFrame_Ph2::lastUnsaturatedSample() const {
```

Comment:
This algorithm probably needs changing for phase2.

Reply:
Yes. For the moment, we do not use this algorithm at all for Phase2. We are considering how to modify it in such a way that we can use it in the EcalCoder. Otherwise we will remove it completely